app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html

```diff
@@ -114,15 +119,17 @@
                             progressive_disclosure(
                                 id = s"activity-events-row-$index",
                                 summary = summary,
-                                body = body.comment.map(Html(_))
+                                body = body.comment.map(Html(_)),
+                                Some(startAtTabIndex + (index*2))
                             )
                         }
                         case EventType.QUEUE_CHANGE => {
                             val body = event.details.asInstanceOf[QueueChange]
                             progressive_disclosure(
                                 id = s"activity-events-row-$index",
                                 summary = s" ${assignee(event.operator, true)} moved this case to the ${body.to.flatMap(to => queues.find(to == _.id)).map(_.name).getOrElse("unknown")} queue",
-                                body = body.comment.map(Html(_))
+                                body = body.comment.map(Html(_)),
+                                Some(startAtTabIndex + (index*2))
```

Comment:
is that (Some(start...) the tab index property?  If so, why is it not labelled?  "useTabIndex = Some...."

Reply:
good point, will fix